package.json

```diff
@@ -85,6 +85,7 @@
     ]
   },
   "dependencies": {
-    "@types/json-schema": "^7.0.11"
+    "@types/json-schema": "^7.0.11",
+    "json-schema-traverse": "^1.0.0"
```

Comment:
It should be installed as dev dependency.

Reply:
ups, actually, it should not be there but in the `package.json` from `tools/bundler`. Thanks good catch!